spec/helpers/session.rb

```diff
@@ -0,0 +1,33 @@
+module SessionHelpers
+
+  def sign_up (name: 'Edward Kerry',
+              user_name: 'Edward',
+              email: 'edwardkerry@gmail.com',
+              password: 'password123',
+              password_confirmation: 'password123')
+
+    visit '/'
+    click_button 'Sign Up'
+    fill_in :name, with: name
+    fill_in :user_name, with: user_name
+    fill_in :email, with: email
+    fill_in :password, with: password
+    fill_in :password_confirmation, with: password_confirmation
+    click_button 'Submit'
+  end
+
+  def sign_in (email: 'edwardkerry@gmail.com',
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)